Source/WarpX.cpp

```diff
@@ -666,14 +666,15 @@ WarpX::ReadParameters ()
             }
         }
 
-        pp_warpx.query("do_pml", do_pml);
+        if ( pp_warpx.query("do_pml", do_pml) )
```

Comment:
This should go in the `BackwardCompatibility` routine.

Reply:
Thank you @dpgrote ! fixed this now